conf/application.conf

```diff
@@ -264,6 +272,12 @@ Test {
     enabled = true
   }
 
+  deleteUnusedApplicationsJob {
```

Comment:
I don't think we need this as these are the default values in the provider anyway.